server/middlewares/authenticate.js

```diff
@@ -0,0 +1,20 @@
+const jwt = require('jsonwebtoken');
+
+const authenticate = function (req, res, next) {
+  try {
+    const token = req.headers.token;
+    const decoded = jwt.verify(token, process.env.SECRET);
```

Comment:
verify didefine functionnya di helpers juga, biar require jwtnya cuma sekali
```suggestion
const verify = require('../helpers/jwt').verify;

const authenticate = function (req, res, next) {
  try {
    const token = req.headers.token;
    const decoded = verify(token, process.env.SECRET);
```